homeassistant/components/nissan_leaf/device_tracker.py

```diff
@@ -0,0 +1,46 @@
+"""Support for tracking a Nissan Leaf."""
+import logging
+
+from homeassistant.components.nissan_leaf import (
+    DATA_LEAF, DATA_LOCATION, SIGNAL_UPDATE_LEAF)
+from homeassistant.helpers.dispatcher import dispatcher_connect
+from homeassistant.util import slugify
+
+_LOGGER = logging.getLogger(__name__)
+
+DEPENDENCIES = ['nissan_leaf']
+
+ICON_CAR = "mdi:car"
+
+
+def setup_scanner(hass, config, see, discovery_info=None):
+    """Set up the Nissan Leaf tracker."""
+    _LOGGER.debug("Setting up Scanner (device_tracker) for Nissan Leaf, "
+                  "discovery_info=%s", discovery_info)
+
+    def see_vehicle():
+        """Handle the reporting of the vehicle position."""
+        for key, value in hass.data[DATA_LEAF].items():
+            host_name = value.leaf.nickname
+            dev_id = 'nissan_leaf_{}'.format(slugify(host_name))
+            if not value.data[DATA_LOCATION]:
+                _LOGGER.debug("No position found for vehicle %s", key)
+                return False
```

Comment:
Nothing is checking this return value.

Reply:
Fixed with commit https://github.com/filcole/home-assistant/commit/341b9475147ef483fe429d1550dc2520b34a7d6b